lib/backend/model/bgppeer.go

```diff
@@ -34,8 +35,9 @@ var (
 )
 
 type NodeBGPPeerKey struct {
-	Nodename string `json:"-" validate:"omitempty"`
-	PeerIP   net.IP `json:"-" validate:"required"`
+	Nodename            string `json:"-" validate:"omitempty"`
+	PeerIP              net.IP `json:"-" validate:"required"`
+	KeepOriginalNextHop bool   `json:"-" validate:"omitempty"`
```

Comment:
Unless we're doing something very odd downstream, I don't think this belongs in the key for this resource